src/lemmas.k.md

```diff
@@ -75,20 +75,6 @@ library to the logical `#rangeUInt` conditions expressed within the specs.
 rule A -Word B <=Int A => #rangeUInt(256, A -Int B)
   requires #rangeUInt(256, A)
   andBool #rangeUInt(256, B)
-
-// add
-rule (chop(X +Int Y) >=Int X) => #rangeUInt(256, X +Int Y)
```

Comment:
When we have `iff in range uint256 \ X + Y`, there would be no `chop` by the time we get to the `add`.

Is this here for the fail spec?